partuniverse/static/all/jquery-tablesort/index.html

```diff
@@ -0,0 +1,143 @@
+<html>
```

Comment:
i copied the complete jquery-tablesort folder because it was done also with the existing stuff in /static